backend/app.js

```diff
@@ -1,24 +1,11 @@
-const express = require('express')
-const bodyParser = require('body-parser')
-const cors = require('cors')
-const app = express()
+const express = require('express');
+const app = express();
 
-app.use(cors())
```

Comment:
We need these lines for the API to work.

Reply:
The routers attached to this guy will have it if it's needed

Reply:
Dang, that's a lot of code reuse.